### ISSUE TITLE
don't auto expand categories

### ISSUE THREAD

**Issue by pcause** [martinrotter/rssguard#283]
in various places you auto expand categories.  i use categories because for many categories i have a lot of feeds.  it is annoying to have them auto-expanded.  can there be an option to never auto-expand categories.  The "show feed branches" check box disables completely and i have to uncheck this.  leave the + sign but make that checkbox a "don't auto-expand tree branches"

**Comment by martinrotter** [comment]
COuld you please re-phrase " The "show feed branches" check box disables completely and i have to uncheck this. leave the + sign but make that checkbox a "don't auto-expand tree branches"". Not sure if I understood this one correctly.

**Comment by pcause** [comment]
I think the "show feed branches" is the wrong functionality.  The issue is really whether or not you auto-expand branches.  If all branches start collapsed and the user never clicks the "+" sign next to the top element of the branch you never see children.  But it is often the case you want to view a part of the tree expanded (categories) but not that parts children, except occasionally.  I feel a better way to handle the use case is to:

A) Always start with tree branches collapsed
B)  Change the "show feed branches" to be "auto-expand tree branches".  Behavior described below
C) if the user selects an item with children the branch is expanded if auto-expand is set and is not expanded if auto-expand is not set
D) Clicking the + next to an unexpanded item with children expands the branch
E) For any expanded item, clicking the - next to the item collapses that branch
F) state of tree branches is remembered between restarts would be nice

**Comment by martinrotter** [comment]
1. `Show feed branches` is purely VISUAL/COSMETIC feature. Literally the only thing this checkbox does is to display (or hide) arrows which indicates that the item in feed's list has some child items (for example category contains some child feeds).
2. If you have `Show feed branches` feature disabled, then arrows are not visible, but you can expand category items by double-clicking them with your mouse or pushing right arrow button on your keyboard (when the item is selected).
3. RSS Guard remembers expand states for all categories. So if you collapse your category and restart RSS Guard, then your category will be of course collapsed.
4. The only item in feed's list which is automatically expanded when RSS Guard starts is the "Labels" item. No other items are auto-expanded.
5. Pushing `+` or `-` keyboard buttons with exandable item already works as expected and expands or collapses the item.

So, basically, from what you wrote, it only makes sense for me to implement feature "Auto-expand item when selected" which will expand item when selected with keyboard/mouse. Does it make sense?

**Comment by martinrotter** [comment]
https://github.com/martinrotter/rssguard/issues/291
